waspc/src/Wasp/Generator/ServerGenerator/Common.hs

```diff
@@ -99,3 +102,14 @@ dotEnvServer = [relfile|.env.server|]
 -- client SDK identifiers. Useful when creating `context.entities` JS objects in Wasp templates.
 entityNameToPrismaIdentifier :: String -> String
 entityNameToPrismaIdentifier entityName = toLower (head entityName) : tail entityName
+
+buildEntityData :: String -> Aeson.Value
+buildEntityData name =
+  object
+    [ "name" .= name,
+      "prismaIdentifier" .= entityNameToPrismaIdentifier name
+    ]
+
+-- https://github.com/wasp-lang/wasp/issues/812#issuecomment-1335579353
+toESModulesImportPath :: FilePath -> FilePath
+toESModulesImportPath = (++ ".js") . reverse . drop 3 . reverse
```

Comment:
I'm not an expert by no means, but I know you can use `takeBaseName` from `System.FilePath.Posix` to get the base name from a path e.g. `takeBaseName "@ext/folder/name.js" == "name"`

Reply:
This function is not clear about what it does!
I agree that splitExtension is probably better choice, although I can't be for sure until I understand what we are trying to do here. Interesting you are always dropping 3 chars -> so you expect file that has extension of length 2?
Anyway, let's put some comments on it -> GH issue link is nice, but as a supportive thing, not as mandatory reading to understand what function does.
Maybe we could also improve the name.

Reply:
Btw would it be candidate for using StrongPath? I believe we expect paths to be relative? It doesn't have to though, just wondering, depends if that brings some value or not.